front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarProjectsItem.tsx

```diff
@@ -14,29 +14,32 @@ interface Props {
   connectDropTarget: any;
```

Comment:
Could this have a better type?

Reply:
Probably, but I'm sure this will be fixed by other work (either upgrading the library or working towards strict mode), so not going to do this now.